lib/models/room.js

```diff
@@ -261,6 +280,29 @@ Room.prototype.addEvents = function(events, duplicateStrategy) {
 };
 
 /**
+ * Removes events from this room.
+ * @param {String} event_ids A list of event_ids to remove.
+ */
+Room.prototype.removeEvents = function(event_ids) {
+    // avoids defining a function in the loop, which is a lint error
+    function remove_event_with_id(timeline, id) {
+        // NB. we supply reverse to search from the end,
+        // on the assumption that recents events are much
+        // more likley to be removed than older ones.
+        return utils.removeElement(timeline, function(e) {
+            return e.getId() == id;
+        }, true);
+    }
+
+    for (var i = 0; i < event_ids.length; ++i) {
+        var removed = remove_event_with_id(this.timeline, event_ids[i]);
+        if (removed !== false) {
```

Comment:
Is this right? AFAIK `removeElement` now returns the element that was removed, which will be a `MatrixEvent`. This will always return `false`. Did you intend to `Boolean()` before returning in `remove_event_with_id`?